website/default_config.py

```diff
@@ -25,24 +25,24 @@
 # The Picard developers have standardized on using only 'dev' or 'final' as the str_type segment of the version tuple.
 PICARD_VERSIONS = {
     'stable': {
-        'tag': '2.9.2',
-        'version': (2, 9, 2, 'final', 0),
+        'tag': '2.10.0',
```

Comment:
I think this must be only 2.10, as in https://github.com/metabrainz/picard-website/commit/c02011973a76f18c595d6363a391f7aa7da08f2d

Reply:
Good catch, fixed in [058f9df](https://github.com/metabrainz/picard-website/pull/285/commits/058f9df54c81197e06d69e196d25a2f90ee6f371)